.github/workflows/build-mac-github-release-artifacts.yaml

```diff
@@ -1,9 +1,6 @@
-name: "release-mac"
+name: "Build Mac Github Release Artifacts"
```

Comment:
uber nit, since we're renaming these files do you mind also adding the arch in the filename like `mac-m1` / `mac-intel`? I get confused between the "mac" and "m1" files sometimes since m1 is also mac :sweat_smile: